pkg/3scale/amp/component/system_options_builder.go

```diff
@@ -290,6 +281,9 @@ func (s *SystemOptionsBuilder) setRequiredOptions() error {
 		return fmt.Errorf("no wildcard domain has been provided")
 	}
 
+	if s.options.s3FileStorageOptions != nil && s.options.pvcFileStorageOptions != nil {
```

Comment:
Should we have this validation here? Should we assume the user of this class will "use it correctly" and therefore is not needed?